src/__tests__/src/context/GameContextHelper.test.ts

```diff
@@ -0,0 +1,32 @@
+import { winner } from "../../../const/gameWinner";
+import gameContextHelpers from "../../../context/GameContextHelper";
+
+test("should be draw", () => {
+  expect(gameContextHelpers.findWhoWonRound(21, 21)).toBe(winner.DRAW);
```

Comment:
myślę że spokojnie 2-3 skrajne przypadki by wystarczyły, niepotrzebnie zamulamy testy. Jak już byś chciał być bardzo dokładny to warto by np w pętli sprawdzić wszystkie możliwości. Jest ma wbudowaną funkcję each: https://jestjs.io/docs/api#testeachtablename-fn-timeout